tests/__snapshots__/config@eslint-config-airbnb

```diff
@@ -12,6 +12,7 @@ const compat = new FlatCompat({baseDirectory: __dirname, recommendedConfig: plug
 
 /** @type {import('eslint').Linter.Config[]} */
 export default [
+  {ignores: ["**/*.js", "**/*.cjs", "**/*.mjs"]},
```

Comment:
Tests should be updated to pass `languages: ["javascript"]`?

Reply:
👍

Reply:
cabfdc90cedcef655dc02acefe496acb3732b1d3